__integrations__/models/profile.test.ts

```diff
@@ -445,4 +445,37 @@ describe("Models |> Profile", () => {
       context.dispose();
     }
   );
+
+  it.each`
+    inputValue    | expectedValue
+    ${undefined}  | ${"UNSET"}
```

Comment:
isn't `null` case missing?

Reply:
I talked with @BurnedMarshal and we decided that null is an incoherent case that won't happen at this stage.